<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormApiClimaTempo</class>
 <widget class="QWidget" name="FormApiClimaTempo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>540</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Consulta Clima - API Clima Tempo</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Consulta do clima atrav√©s da Api do Clima Tempo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonConsultaCidade">
       <property name="text">
        <string>Consultar Cidade</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButtonRegistraId">
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Registrar Id</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditConsultaCidade">
       <property name="placeholderText">
        <string>Digite a cidade</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditResitraId">
       <property name="placeholderText">
        <string>Digite o id</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonConsulaTempo">
       <property name="text">
        <string>Consultar Tempo Atual</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="textEditCidadeId">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>150</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="textEditTempo">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>300</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditConsultaCidade</tabstop>
  <tabstop>pushButtonConsultaCidade</tabstop>
  <tabstop>lineEditResitraId</tabstop>
  <tabstop>pushButtonRegistraId</tabstop>
  <tabstop>pushButtonConsulaTempo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
